build/contracts/AZTECERC20Bridge.json

```diff
@@ -8637,1518 +9097,1119 @@
-                                  "typeIdentifier": "t_tuple$__$",
-                                  "typeString": "tuple()"
-                                }
-                              },
-                              "id": 370,
-                              "nodeType": "ExpressionStatement",
-                              "src": "9533:123:1"
-                            }
-                          ]
+                  "value": null,
+                  "visibility": "internal"
+                },
+                {
+                  "constant": false,
+                  "id": 287,
+                  "name": "challenge",
+                  "nodeType": "VariableDeclaration",
+                  "scope": 341,
+                  "src": "7500:14:1",
+                  "stateVariable": false,
+                  "storageLocation": "default",
+                  "typeDescriptions": {
+                    "typeIdentifier": "t_uint256",
+                    "typeString": "uint256"
+                  },
+                  "typeName": {
+                    "id": 286,
+                    "name": "uint",
+                    "nodeType": "ElementaryTypeName",
+                    "src": "7500:4:1",
+                    "typeDescriptions": {
+                      "typeIdentifier": "t_uint256",
+                      "typeString": "uint256"
+                    }
+                  },
+                  "value": null,
+                  "visibility": "internal"
+                },
+                {
+                  "constant": false,
+                  "id": 289,
+                  "name": "domainHashT",
+                  "nodeType": "VariableDeclaration",
+                  "scope": 341,
+                  "src": "7521:19:1",
+                  "stateVariable": false,
+                  "storageLocation": "default",
+                  "typeDescriptions": {
+                    "typeIdentifier": "t_bytes32",
+                    "typeString": "bytes32"
+                  },
+                  "typeName": {
+                    "id": 288,
+                    "name": "bytes32",
+                    "nodeType": "ElementaryTypeName",
+                    "src": "7521:7:1",
+                    "typeDescriptions": {
+                      "typeIdentifier": "t_bytes32",
+                      "typeString": "bytes32"
+                    }
+                  },
+                  "value": null,
+                  "visibility": "internal"
+                }
+              ],
+              "src": "7445:99:1"
+            },
+            "payable": false,
+            "returnParameters": {
+              "id": 291,
+              "nodeType": "ParameterList",
+              "parameters": [],
+              "src": "7562:0:1"
+            },
+            "scope": 380,
+            "src": "7419:1288:1",
+            "stateMutability": "nonpayable",
+            "superFunction": null,
+            "visibility": "internal"
+          },
+          {
+      
```

Comment:
Can we remove changes to the `build` folder from this pull request? The new contracts don't have deployed versions on mainNet and the test-nets. Until we've improved our deployment pipeline we need the contracts in `build` to point to their deployed counterparts.